authors/wsgi.py

```diff
@@ -11,6 +11,6 @@
 
 from django.core.wsgi import get_wsgi_application
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "authors.settings")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "authors.settings.development")
```

Comment:
This still caters only for development environment, I think it would be better to have a conditional that caters for both development and production.
Do the same for your **manage.py** module